### ISSUE TITLE
CONCEPT_CLASS_ID = 'Undefined' increased for SNOMED

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Vocabulary-v5.0#177]
```
SELECT *
FROM CONCEPT
WHERE CONCEPT_CLASS_ID = 'Undefined'
AND VOCABULARY_ID = 'SNOMED'
AND (INVALID_REASON IS NULL OR INVALID_REASON = '')
```

Noticed an increase from 56 --> 396 SNOMED concepts that now have a CONCEPT_CLASS_ID = 'Undefined'.  I'm bringing it up for you to just double check that the increase was warranted.

**Comment by dimshitc** [comment]
Hi @
Thanks for reporting.
Does it affects some of your projects? I mean should it be fixed urgently?
The thing is that we are in the middle of adding UK Drug Extension to SNOMED, so refresh of SNOMED might take some time

**Comment by ericaVoss** [comment]
I just noticed it and thought to tell you, doesn't impact me.  I'm just doing my normal testing.  So no urgency on my side.

**Comment by cgreich** [comment]
@ericaVoss: thanks for doing this.

**Comment by dimshitc** [comment]
there are 3 of them only now

concept_id | concept_name
-- | --
36676866 | Supplier
36676867 | Product name
42538930 | OWL metadata concept

Can we just leave them 'Undefined' forever?:)

**Comment by cgreich** [comment]
@dimshitc, @hardhouse 

"Undefined" is a temporary domain used in load_stage. It was supposed to be there for the only purpose of throwing an extension, because the vocab team forgot to assign each new SNOMED concept that doesn't fall under a "peak". What happened? Did you switch off a QA test again?

**Comment by dimshitc** [comment]
We really didn't know how to name the class of these concepts, so left them Undefined. 
Note, It's concept_class_id, not domain.
But anyway, "Undifined" should be reserved to some kind of logic missing, while these can have concept_class_id = 'Metadata'. Agree?

**Comment by cgreich** [comment]
Agreed. All normal proper concepts should have a concept_class_id and domain_id. I know sometimes it is hard.

**Comment by TinyRickC137** [comment]
Fixed in all proper concepts
